python/src/opendp/extras/numpy/__init__.py

```diff
@@ -205,6 +205,16 @@ class NPArray2Domain(NamedTuple):
     cardinalities: numpy.ndarray | None
     T: str | RuntimeType
 
+# Without these we get: "Alias for field number ..."
+NPArray2Domain.origin.__doc__ = 'center of the norm region'
+NPArray2Domain.norm.__doc__ = 'each row in x is bounded by the norm'
+NPArray2Domain.p.__doc__ = 'designates L`p` norm'
+NPArray2Domain.size.__doc__ = 'number of rows in data'
+NPArray2Domain.num_columns.__doc__ = 'number of columns in the data'
+NPArray2Domain.nan.__doc__ = 'whether NaN values are allowed'
+NPArray2Domain.cardinalities.__doc__ = 'cardinalities of the categorical columns'
+NPArray2Domain.T.__doc__ = 'atom type'
```

Comment:
Does typing.NamedTuple subclassing allow you to put docstrings on the attributes themselves, like how dataclasses work? Maybe we should consider switching everything to dataclasses, for consistency.


```suggestion

class NPArray2Domain(NamedTuple):
    origin: numpy.ndarray | None
    """center of the L`p` ball enclosing each row"""
    norm: float | None
    """radius of the L`p` ball enclosing each row"""
    p: Literal[1, 2, None]
    """designates L`p` norm"""
    size: int | None
    """number of rows in the data"""
    num_columns: int | None
    """number of columns in the data"""
    nan: bool
    """whether NaN values are allowed"""
    cardinalities: numpy.ndarray | None
    """cardinalities of categorical columns"""
    T: str | RuntimeType
    """atom type"""
```

Reply:
That doesn't seem to work: I'm still getting 
```
 |  norm
 |      Alias for field number 1
```

I'll try switching to dataclasses...